.gitignore

```diff
@@ -7,3 +7,5 @@
 *.scss.d.ts
 # Dependency directories
 node_modules/
+.DS_STORE
```

Comment:
I just figured them out with Mac as my primary machine. This is something that mac machines produce if you interact with `Finder` on terminal.